generators/client/templates/angular/package.json.ejs

```diff
@@ -157,6 +158,11 @@
     "workbox-webpack-plugin": "4.3.1",
     "write-file-webpack-plugin": "4.5.1"
   },
+<%_ if (clientPackageManager === 'yarn') { _%>
```

Comment:
Do you require this as you have included that also as a dependency? Either case, I think we should also add a comment to the corresponding issue that we can track to update/remove this block later on.

Reply:
Yes, it is needed for Yarn, for a reason I don't remember
No, we can't add comment in JSON file :D

Reply:
we can always add a comment in generator that's used to include this code :smile:

Reply:
Oh indeed, nice tips !

Reply:
@vishal423, without this `resolve` part Yarn didn't resolve Terser to version 4.2.1, so different config for Npm and Yarn was needed to work correctly.